### ISSUE TITLE
mariadbd: Retry in 60 secs. 

### ISSUE THREAD

**Issue by ejohb** [lmagyar/homeassistant-addon-mariadb-inmemory#12]
Hi.

My recorder/log/history is broken and I've started seeing a log full of the below. I don't think I changed anything. Does anyone know what might be wrong?

```
2022-06-13 12:39:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 12:49:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 12:59:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 13:09:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 13:19:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 13:29:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 13:39:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 13:49:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 13:59:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 14:09:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 14:19:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 14:29:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 14:39:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 14:49:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 14:59:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 15:09:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 15:19:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 15:29:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
2022-06-13 15:39:12 22 [Warning] mariadbd: Retry in 60 secs. Message reprinted in 600 secs
```

**Comment by lmagyar** [comment]
I think this is the result of this issue: https://github.com/home-assistant/core/issues/72883

Aren't there `ERROR (Recorder) [homeassistant.components.recorder.core] Database error during schema migration` errors previously in core log?

TLDR: core 2022.6.0 DB schema migration fails due to a hard-coded statement. 2022.6.1+ should upgrade correctly.

If this is the situation and the DB migration is failed, please upgrade the in-memory DB add-on, it creates the latest DB schema itself, so no migration, no error.

Is this the situation? Add-on update helped?

If not, please copy more core and add-on logs.

**Comment by lmagyar** [comment]
If this is an out-of-space issue, please see the Documentation page, the tmpfs.size and the HA configuration sections. 

Eg. my recorder config is below. It runs every hour to purge DB as soon as possible, because deleting (purge+repack) 24h recorder data needs more temp space. I'm not a mariadb guru, just a power-user, if someone has better idea how to handle the repack-needs-lot-of-temp-space issue, I'm open to any good idea.

```
recorder:
  db_url: !secret recorder_db_url
  auto_purge: false
  exclude:
    event_types:
      - call_service

automation:
  - alias: Auto purge with repack
    mode: queued
    trigger:
      # - platform: time
      #   at: "04:12:00"
      - platform: time_pattern
        minutes: "12"
    action:
      - service: recorder.purge
        data:
          keep_days: 1
          repack: true
```
